frontend/app/views/event/page.scala.html

```diff
@@ -49,7 +49,12 @@ <h1 class="event-masthead__name qa-event-detail-name">@event.name.text</h1>
                     }
                 </div>
             }
-            <img class="event-header__logo" src="@Asset.at("images/providers/" + event.providerOpt.getOrElse(event.metadata.identifier) + ".svg")">
+            @if(event.providerOpt.getOrElse(event.metadata.identifier) == "5x15") {
```

Comment:
IDK, some kind of find or filter maybe `EBEvent.providerWhitelist.find(_._1 == provider)`. Probably OK to leave it as is for now.